unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -1,34 +1,56 @@
 using DCL.Helpers;
 using Newtonsoft.Json;
 using System.Collections.Generic;
+using UnityEngine;
 
 namespace DCL.EquippedEmotes
 {
     /// <summary>
     /// Plugin feature that initialize the Equipped Emotes feature.
     /// </summary>
-    public class EquippedEmotesPlugin : IPlugin
+    public class EquippedEmotesInitializerPlugin : IPlugin
     {
+        internal const string EMOTES_CUSTOMIZATION_FEATURE_FLAG = "emotes_customization";
         internal const string PLAYER_PREFS_EQUIPPED_EMOTES_KEY = "EquippedNFTEmotes";
 
         internal DataStore_EmotesCustomization emotesCustomizationDataStore => DataStore.i.emotesCustomization;
+        internal DataStore_FeatureFlag featureFlagsDataStore => DataStore.i.featureFlags;
 
-        public EquippedEmotesPlugin()
+        public EquippedEmotesInitializerPlugin()
         {
             LoadDefaultEquippedEmotes();
 
-            emotesCustomizationDataStore.isEmotesCustomizationInitialized.OnChange += OnEmotesCustomizationInitialized;
-            OnEmotesCustomizationInitialized(emotesCustomizationDataStore.isEmotesCustomizationInitialized.Get(), false);
+            featureFlagsDataStore.flags.OnChange += OnFeatureFlagsChanged;
+            OnFeatureFlagsChanged(featureFlagsDataStore.flags.Get(), null);
 
-            emotesCustomizationDataStore.equippedEmotes.OnSet += SaveEquippedEmotesInLocalStorage;
+            emotesCustomizationDataStore.equippedEmotes.OnSet += OnEquippedEmotesSet;
+            emotesCustomizationDataStore.equippedEmotes.OnAdded += OnEquippedEmoteAddedOrRemoved;
+            emotesCustomizationDataStore.equippedEmotes.OnRemoved += OnEquippedEmoteAddedOrRemoved;
         }
 
-        private void OnEmotesCustomizationInitialized(bool current, bool previous)
+        internal List<string> GetDefaultEmotes()
+        {
+            return new List<string>
+            {
+                "handsair",
+                "wave",
+                "fistpump",
+                "dance",
+                "raiseHand",
+                "clap",
+                "money",
+                "kiss",
+                "headexplode",
+                "shrug"
+            };
+        }
+
+        internal void OnFeatureFlagsChanged(FeatureFlag current, FeatureFlag previous)
         {
-            if (!current)
+            if (!current.IsFeatureEnabled(EMOTES_CUSTOMIZATION_FEATURE_FLAG))
```

Comment:
I wouldnt unsubscribe so we automatically support with runtime changes on feature flags.

```
           if (current.IsFeatureEnabled(EMOTES_CUSTOMIZATION_FEATURE_FLAG))
                LoadEquippedEmotesFromLocalStorage();
           else
                LoadDefaultEquippedEmotes();
```